From c24a1c0daf2b7f515420385bcd0f2c9c38d62bf7 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Mon, 15 Jun 2015 10:57:24 +0800
Subject: [PATCH] MLK-11109-1 mxc IPUv3: PRE reg: Correct store engine status
 block Y mask

This patch corrects the macro BM_PRE_STORE_ENGINE_STATUS_STORE_BLOCK_Y from
0xFFFF0000 to 0x3FFF0000 according to the i.MX6QP reference manual.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 drivers/mxc/ipu3/pre-regs.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mxc/ipu3/pre-regs.h b/drivers/mxc/ipu3/pre-regs.h
index ee5d424..5b2cf93a 100644
--- a/drivers/mxc/ipu3/pre-regs.h
+++ b/drivers/mxc/ipu3/pre-regs.h
@@ -454,7 +454,7 @@
 #define HW_PRE_STORE_ENGINE_STATUS	(0x00000120)
 
 #define BP_PRE_STORE_ENGINE_STATUS_STORE_BLOCK_Y      16
-#define BM_PRE_STORE_ENGINE_STATUS_STORE_BLOCK_Y 0xFFFF0000
+#define BM_PRE_STORE_ENGINE_STATUS_STORE_BLOCK_Y 0x3FFF0000
 #define BF_PRE_STORE_ENGINE_STATUS_STORE_BLOCK_Y(v) \
 	(((v) << 16) & BM_PRE_STORE_ENGINE_STATUS_STORE_BLOCK_Y)
 #define BP_PRE_STORE_ENGINE_STATUS_STORE_BLOCK_X      0
-- 
1.8.0

